pkgs/by-name/he/hexapdf/package.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, bundlerApp, bundlerUpdateScript, ruby, makeWrapper, git, docutils, nixosTests }:
+
+bundlerApp rec {
+  pname = "hexapdf";
+  exes = [ "hexapdf" ];
+
+  inherit ruby;
+  gemdir = ./.;
+
+  passthru.updateScript = bundlerUpdateScript "hexapdf";
+
+  meta = with lib; {
+    description = "Versatile PDF creation and manipulation library";
+    homepage = "https://hexapdf.gettalong.org/";
+    changelog = "https://github.com/gettalong/hexapdf/blob/master/CHANGELOG.md";
+    license = licenses.agpl3Only;
```

Comment:
I am uncertain, whether this is correct.
It is released under AGPL Open Source, a commercial licence available (see also https://gettalong.at/hexapdf/).
Do we need to include the commercial option here?

Related links:
- Website about commercial license: https://gettalong.at/hexapdf/
- Custom LICENSE file in source code repository: https://github.com/gettalong/hexapdf/blob/master/LICENSE
- official website: https://hexapdf.gettalong.org/

Reply:
The Gem contains [1](https://github.com/gettalong/hexapdf/blob/e784ea875a60e04512bcbb28fd99c79044ad9941/hexapdf.gemspec#L33)

```ruby
s.licenses = ['AGPL-3.0', 'Nonstandard']
```

so without overseeing possible consequences, I'd propose 

```nix
license = with licenses; [ agpl3Only unfree ];

Reply:
The website states:
> Everything that is available in the commercial version is also available in the AGPL version

So I would say agpl3Only is fine.